runtime/flang/CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ enable_language(C ASM Fortran) # Enable assembly and Fortran
 SET(ASM_OPTIONS "-DLINUX_ELF")
 SET(CMAKE_ASM_FLAGS "${CFLAGS} ${ASM_OPTIONS}" )
 SET(CMAKE_SHARED_LINKER_FLAGS "-no-flang-libs")
+if (${TARGET_ARCHITECTURE} STREQUAL "aarch64")
+  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z -Wl,combreloc")
+  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z -Wl,stack-size=536870912")
```

Comment:
This looks like a phone number. Perhaps a comment would be good here.

Reply:
536870912 bytes == 512 MB.

For OpenMP programs, including SPEC benchmarks, VERY LARGE stack sizes are required. For example, OMP_NUM_THREADS == 64 with OMP_STACKSIZE == 128M.

-Wl,-z -Wl,stack-size=<N> does not set the stack size. It only creates a permissible upper bound, which may or may not be in effect at run-time.

Reply:
Thanks for the explanation.  What's the downside of not including this option in the library build?  It seems odd to hard-code in SPEC benchmark requirements.  In any case, if the line must stay please document it in the CMakeLists.txt file.

Reply:
The downside is that any OpenMP program that has large stack size memory requirements might fail, independently of SPEC.

Very large stack sizes for OpenMP programs is common-place, and is unrelated to SPEC.